RSSTimeFormatter/RealDateTimeFormatter.cs

```diff
@@ -100,131 +82,192 @@ public string PrintTimeCompact(double time, bool explicitPositive)
 		{
 			if (IsInvalidTime(time))
 				return InvalidTimeStr(time);
-			bool isNegativeTime = false;
-			if (time < 0) {
-				time = Math.Abs(time);
-				isNegativeTime = true;
-			}
-			DateTime epoch = GetEpoch();
-			DateTime target = epoch.AddSeconds(time);
-			TimeSpan span = target - epoch;
-			return string.Format("{0}{1}{2:D2}:{3:D2}:{4:D2}"
-				, isNegativeTime ? "- " : (explicitPositive ? "+ " : "")
+			bool isTimeNegative = time < 0;
+			time = Math.Abs(time);
+			TimeSpan span = TimeSpan.FromSeconds(time);
+			return string.Format("{0}{1}{2}"
+				, isTimeNegative ? "- " : (explicitPositive ? "+ " : "")
 				, (span.Days != 0 ? span.Days.ToString() : "")
-				, span.Hours
-				, span.Minutes
-				, span.Seconds
+				, SpanAs24HWithSeconds(span)
 			);
 		}
+
 		public string PrintDateDelta(double time, bool includeTime, bool includeSeconds, bool useAbs)
 		{
 			if (IsInvalidTime(time))
 				return InvalidTimeStr(time);
-			if (time < 0 && useAbs)
-				time = Math.Abs(time);
+			time = useAbs ? Math.Abs(time) : time;

```

Comment:
I really dislike the use of this style conditional.  I know, its personal taste, but I prefer clear verbose `if` statements

Reply:
>  I prefer clear verbose if statements
I do to! I've changed it to:
`if (useAbs) time = Math.Abs(time);`

the `time < 0` is unnecessary because Math.Abs() effectively doesn't do anything if time > 0.

Reply:
Fair enough but could still do it with an if :)

Reply:
That is definitely down to your personal taste. I do agree that the inline conditionals can get messy in some cases but this one isn't the case.